test/models/onnx/dynamic_shapes/atanh_dyn_shape.prototxt

```diff
@@ -0,0 +1,39 @@
+ir_version: 3
+producer_name: "nGraph ONNX Importer"
+graph {
+  node {
+    input: "x"
+    output: "y"
+    op_type: "Atanh"
+  }
+  name: "atanh_graph"
+  input {
+    name: "x"
+    type {
+      tensor_type {
+        elem_type: 1
+        shape {
+          dim {
+            dim_param: "dynamic"
+          }
+          dim {
+            dim_param: "dynamic"
```

Comment:
If I understand the meaning of this label correctly in the ONNX spec, then you are defining a shape with the constraint that the 2 dimentions are equal. As in `dynamic`x`dynamic`, 1x1, 2x2, 3x3, etc. You seem to be using it differently in tests. I would suggest to name one of these variables differently.

Reply:
Thanks, done